File: OSVersion/OSVersion/ViewController.m
//
//  ViewController.m
//  OSVersion
//
//  Created by Luis Castillo on 9/26/16.
//  Copyright © 2016 LC. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    //objective-c
    [self checkOSVersion];
    
    //swift
    [[[CheckOSOversion alloc]init] checkVersion];
}//eom

-(void)checkOSVersion
{
    NSString * osVersionString = [[NSProcessInfo processInfo] operatingSystemVersionString];
    
    NSOperatingSystemVersion osVersion = [[NSProcessInfo processInfo]operatingSystemVersion];
    
    NSInteger maxVersion = osVersion.majorVersion;
    NSInteger minVersion = osVersion.minorVersion;
    
    NSString * printString = @"Objective-c \n";
    printString = [printString stringByAppendingFormat:@" OS (Max.Min) = %ld . %ld  \n" ,(long)maxVersion, (long)minVersion];
    printString = [printString stringByAppendingString:osVersionString];
    
    NSLog(@"%@ \n\n", printString);
}//eom

@end

File: Indicators/Indicators/ViewController.swift
//
//  ViewController.swift
//  Indicators
//
//  Created by Luis Andres Castillo Hernandez on 11/22/15.
//  Copyright © 2015 DevBros. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var indicator: UIActivityIndicatorView!
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.

    }


    @IBAction func startIndicator(_ sender: AnyObject) {
        self.indicator.startAnimating()
        self.indicator.hidesWhenStopped = true
    }

    @IBAction func stopIndicator(_ sender: AnyObject) {
        self.indicator.stopAnimating()
    }
}

